src/config/index.ts

```diff
@@ -26,7 +26,10 @@ export class Config {
     const status = response.responseCode as keyof typeof ErrorMessages;
     let message = "unknown error, please check urway docs for more details";
     if (ErrorMessages[status]) message = ErrorMessages[status];
-    console.log("RESP:", response);
-    throw new Error(`error code ${status}: - ${message}`);
+
+    const exception = new Error();
+    exception.name = status;
```

Comment:
I would suggest extending the Error class with a custom error class that has status as the status code instead of using the name as the status code.

Reply:
Okay, let me check for a way to handle it